src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -300,15 +326,20 @@ function ProviderSelectionField({
             {providersListLength < communityCareProviderList.length && (
               <>
                 <button
+                  aria-atomic="true"
+                  aria-live="assertive"
```

Comment:
Seems like this would need to have `aria-live="polite" role="status" aria-atomic="true"` instead of these two attributes.

Reply:
switched `aria-live="polite"` but cannot add `role="status"` because eslint shows "Interactive elements should not be assigned non-interactive roles."